libs/azure-storage/tests/utils.py

```diff
@@ -84,3 +99,52 @@ def get_test_blobs(
         return [blob for blob in _TEST_BLOBS if blob["blob_name"].startswith(prefix)]
 
     return _TEST_BLOBS
+
+
+def get_datalake_test_blobs(
+    blob_names: Optional[Union[str, Iterable[str]]] = None, prefix: Optional[str] = None
+) -> list[dict[str, Any]]:
+    if blob_names is not None:
```

Comment:
Instead of duplicating the code for `get_test_blobs()`, I'm wondering if we can expose a new `_get_test_blobs()` utility that both `get_test_blobs()` and `get_datalake_test_blobs()` can call out to by passing in the list of blobs to use? Mainly I think this can help reused code especially if we want to expand on the scaffolding in the future.